app/src/components/Modals/add-activity-modal.tsx

```diff
@@ -189,13 +225,8 @@ const AddActivityModal: FC<AddUserModalProps> = ({
                 </FormControl>
                 <FormControl>
                   <BuildingTypeSelectInput
-                    options={[
-                      t("all-fuels"),
-                      t("natural-gas"),
-                      t("propane"),
-                      t("heating-oil"),
-                    ]}
-                    title={t("fuel-type")}
+                    options={formInputs.fields[1].options}
```

Comment:
You are using literal numbers for the indexes on the Fields array. do we have a fixed number of elements here? should this be a loop, or maybe use constants that indicate which field is being shown?

Reply:
Yes we have a fixed number of elements.